File: JagWebApp.Tests/Stubs/DishStub.cs
using JagWebApp.Core.Models;

namespace JagWebApp.Tests.Stubs
{
    class DishStub
    {
        public static Dish GetDish()
        {
            return new Dish
            {
                Id = 1,
                Name = "a",
                Category = new Category { Id = 1, Name = "a" },
                Amount = 2
            };
        }
    }
}

File: JagWebApp/ClientApp/src/test/stubs/dish.stub.ts
import { photoStub } from "./photo.stub";
import { Dish } from "../../app/models/dish";

export const dishStub: Dish = {
    id: 1,
    name: 'a',
    categoryId: 1,
    category: {
        id: 1,
        name: 'b'
        },
    amount: 1,
    mainPhoto: photoStub
}
